### PR TITLE
Add aspect ratio presets support via theme.json

### PR DESCRIPTION
**Pull request by fabiankaegy**
Trac ticket: https://core.trac.wordpress.org/ticket/61239
---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-includes/class-wp-theme-json.php [hunk 1]
```diff
@@ -390,8 +400,10 @@ class WP_Theme_JSON {
 		),
 		'custom'                        => null,
 		'dimensions'                    => array(
-			'aspectRatio' => null,
-			'minHeight'   => null,
+			'aspectRatio'         => null,
+			'aspectRatios'        => null,
+			'defaultAspectRatios' => null,
+			'minHeight'           => null,
```
Comment (tellthemachines) [inline src/wp-includes/class-wp-theme-json.php:407]: This change should probably also get a `@since` annotation for completeness 😄

**Review by tellthemachines — COMMENTED** [review]
Aside from the annotation, changes LGTM and work as expected when following the testing steps in https://github.com/WordPress/gutenberg/pull/47271!

File: src/wp-includes/class-wp-theme-json.php (hunk 1 above)
Comment (fabiankaegy) [reply src/wp-includes/class-wp-theme-json.php:407]: @tellthemachines Done :) Thanks for the review

Comment (tellthemachines) [reply src/wp-includes/class-wp-theme-json.php:407]: Oh sorry, I meant an annotation for this change above the `VALID_SETTINGS` array! Though the addition you made to `PRESETS_METADATA` is also valuable.

Comment (fabiankaegy) [reply src/wp-includes/class-wp-theme-json.php:407]: Ahh my mistake. Just added that also :)

Comment (tellthemachines) [reply src/wp-includes/class-wp-theme-json.php:407]: Awesome, thanks!

**Review by tellthemachines — APPROVED** [review]

**Comment by tellthemachines** [comment]
Committed in [r58242](https://core.trac.wordpress.org/changeset/58242).

**Pull request closed without merging**